src/main/java/com/bootcamp/demo/model/Session.java

```diff
@@ -0,0 +1,49 @@
+package com.bootcamp.demo.model;
+
+import java.time.LocalDateTime;
+
+/**
+ * A Session object stores the user, the start and end date and time of the session
+ * <p>
+ * Next needed update: DI of the User attribute automatically when a session is created, and the user will be
+ * recognised after the Log In
+ *
+ * @author Denisa Dragota
+ * @version 11/11/2021
+ */
+
+public class Session {
+
+    private final LocalDateTime startSession;
```

Comment:
**Just a recommendation / advice**: starting with Java 8, there were some classes introduced that help manipulating the dates and time more easily.

For example, in this case, we could use the class java.time.Instant instead of LocalDateTime.
`private final Instant startSession;`

LocalDateTime is fine as well, can remain, but by using Instant you have some additional features for handling it.

Reply:
Done